donations.markdown

```diff
@@ -0,0 +1,60 @@
+---
+title: Supporting Haskell.org
+page: donations
+isDonations: true
+---
+
+# Support the Community
+
+A significant part of the Haskell community infrastructure—like Hackage, Hoogle, the GHC build infrastructure and this website—runs on donations from the Haskell community. If you would like to support these efforts, you have several options for donating:
```

Comment:
```suggestion
A significant part of the Haskell community infrastructure—like Hackage, Hoogle, the GHC build infrastructure and this website—runs on monetary and in-kind donations from the Haskell community. If you would like to support these efforts, you have several options for donating:
```
Can we do a small tweak to this sentence?  Most of the infra runs on in-kind donations now and Herbert alerted me that not mentioning this is a bit misleading.